### PR TITLE
Add additional design guidelines to the documentation

### PR DESCRIPTION
**Pull request by thehorse14**

### CODE REVIEW

File: chatbot/chatbot-design-guidelines.md [hunk 1]
```diff
@@ -128,4 +128,64 @@ _NOTE: Please [refer here](./setting-up-chatbot-agent-in-giant.md) for the detai
...
+Before: Replies without information <br/><br/>
+![Quick replies claims](./images/chatbot-design-guidelines-images/quick_replies_claims.PNG "quick replies") <br/><br/>
+After: Replies with informative icon and description <br/><br/>
+![Card replies claims](./images/chatbot-design-guidelines-images/card_replies_claims.PNG "card replies") <br/><br/>
+
+- Use different font type to emphasize text. <br/><br/>
+Before: Replies with same font <br/><br/>
+![Text without emphasis](./images/chatbot-design-guidelines-images/text_without_emphasis.PNG "Font emphasis") <br/><br/>
+After: Replies with different font type to emphasize text <br/><br/>
+![Text emphasis](./images/chatbot-design-guidelines-images/text_emphasis.PNG "Font emphasis") <br/><br/>
```
Comment (tchyong) [inline chatbot/chatbot-design-guidelines.md]: picture is not showing. Please check

File: chatbot/chatbot-design-guidelines.md [hunk 2]
```diff
@@ -128,4 +128,64 @@ _NOTE: Please [refer here](./setting-up-chatbot-agent-in-giant.md) for the detai
...
+
+- Use different font color to emphasize text. <br/><br/>
+Before: Replies with same font colour <br/><br/>
+![Text without color emphasis](./images/chatbot-design-guidelines-images/text_without_color_emphasis.PNG "No color emphasis") <br/><br/>
+After: Replies with different font color to emphasize text <br/><br/>
+![Color emphasis](./images/chatbot-design-guidelines-images/color_emphasis.PNG "Color emphasis")
```
Comment (tchyong) [inline chatbot/chatbot-design-guidelines.md]: picture not showing. Please check. Believe is the extension type case sensitivity

**Comment by izairiabdlatif** [comment]
Reviewed the latest version and this is good to go (all images shown as expected)

**Comment by tchyong** [comment]
LGTM

**Pull request merged**